content/blog/granite-upgrade.mdx

```diff
@@ -0,0 +1,202 @@
+---
+title: Avalanche Granite Upgrade - Enhancing ICM, Unlocking Biometric Use Cases, and Enabling Dynamic Block Times
+description: The Avalanche Granite upgrade activates on Fuji Testnet on October 29th, 2025, bringing major improvements including P-Chain epoched views for ICM, secp256r1 support for biometric authentication, and dynamic minimum block times for faster transactions.
+date: 2025-10-21
+authors: [AvaxDevelopers]
+topics: [Granite Upgrade, ICM, Biometric, Block Time, ACP, Performance]
+comments: true
+---
+
+## Introducing Avalanche Granite
+
+A [pre-release](https://github.com/ava-labs/avalanchego/releases/tag/v1.14.0-fuji) for the latest upgrade to the Avalanche Network, dubbed "Avalanche Granite", has been published. The upgrade is scheduled to activate on the **Fuji Testnet at 11 AM ET (3 PM UTC) on Wednesday, October 29th, 2025**. Following successful activation and testing on the Avalanche Fuji Testnet, an Avalanche Mainnet release will be published, with a set future activation time.
+
+The Granite upgrade is driven by three Avalanche Community Proposals that significantly enhance the network's capabilities:
+
+- [ACP-181: P-Chain Epoched Views](https://github.com/avalanche-foundation/ACPs/tree/main/ACPs/181-p-chain-epoched-views)
+- [ACP-204: Precompile for secp256r1 Curve Support](https://github.com/avalanche-foundation/ACPs/tree/main/ACPs/204-precompile-secp256r1)
+- [ACP-226: Dynamic Minimum Block Times](https://github.com/avalanche-foundation/ACPs/tree/main/ACPs/226-dynamic-minimum-block-times)
+
+---
+
+## Node Operator Requirements
+
+### Mainnet Node Operators
+
+**Action Required**: All Mainnet node operators must upgrade their nodes to **v1.14** before the activation time.
+
+The Avalanche Granite Upgrade includes protocol optimizations that are not compatible with AvalancheGo versions < v1.14. If you run a node on Mainnet, you must upgrade your software to AvalancheGo >= v1.14 before the scheduled activation time. Ensure your node is running with the correct plugin version as specified in the [release notes](https://github.com/ava-labs/avalanchego/releases).
```

Comment:
the plugin version is the same for both mainnet and fuji, can we please include the same language we include in the fuji section in the mainnet section as well? cc: @danielgruesso